python/lsst/dax/imgserv/MetadataFitsDb.py

```diff
@@ -50,13 +49,15 @@ def isDateFormatValid(dt):
     except ValueError:
         return False
 
-def executeInsertList(cursor, table, columnValues, logger=log):
+def executeInsertList(conn, table, columnValues, logger=log):
     '''Insert the columnValues into 'table'
     columnValue is a list of column name and value pairs.
     Values are sanitized.
+
+    Returns lastrowid or None
     '''
     if len(columnValues) < 1:
-        return
+        return None
```

Comment:
Technically this is the same thing, but I agree that explicit `None` is better if function is expected to return something.